README.md

```diff
@@ -1 +1,95 @@
-# Portfolio-
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portoflio website</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)<br>
+🛠 Built With<br>
+-Tech Stack<br>
+-Key Features<br>
+💻 Getting Started<br>
+-Setup<br>
+-Prerequisites<br>
```

Comment:
- [ ] Well done adding a README to make a professional readme file; kindly customize the sections, fill the section with the project details for a given milestone, and also look professional[ use markdown ](https://www.markdownguide.org/basic-syntax/)